metaseq/cli/train.py

```diff
@@ -583,6 +588,16 @@ def post_checkpoint_callback(cfg, num_updates, training_finished, filename):
             except (FileNotFoundError, AssertionError) as e:
                 logger.info(f"could not upload {filename}: {e}")
 
+            # TODO[Susan]: Add symlink logic here.
+
+    # if files_to_symlink_to is not None and len(files_to_symlink_to) > 1:
```

Comment:
still need to add logic for 'files_to_symlink_to'. what are you missing exactly from me?

Reply:
Ah nothing needed from you - I need to put this in and see how it works on another cluster too.